scripts/ptp-vrt/test-shim-smoke.sh

```
@@ -0,0 +1,63 @@
+[[ -e "$DEV" ]]
+
+export PTP_VRT_PHC="$DEV"
+export LD_PRELOAD="$ROOT/libptp_vrt_shim.so"
+
+BEFORE=$("$ROOT/test_vrt_cli" gettime)
+"$ROOT/test_vrt_cli" step >/dev/null
+AFTER=$("$ROOT/test_vrt_cli" gettime)
+
+# Host CLOCK_REALTIME should be essentially unchanged by the step (shim only).
+HOST_NOW=$(date +%s)
+BEFORE_SEC=${BEFORE%%.*}
```

Shellcheck (0.11.0)

[warning] BEFORE_SEC appears unused. Verify use (or export if used externally).

(SC2034)